src/public/index.html

```diff
@@ -9,37 +9,54 @@
   </head>
 
 
-    <body>
-      <header>
-        <h1>Something Awesome!</h1>
-      </header>
-      <nav>
-        <ul>
-          <li><a href="/about/">About</a></li>
-          <li> <a href="/contact/">Contact</a></li>
-        </ul>
-      </nav>
+  <body>
+    <header>
+      <div class="title">
+        <h1 class="plaintext-color">
+            Something Awesome! <br/>
+        </h1>
+      </div>
+    </header>
+      <div class="content-container page">
+        <div class="nav-width">
+          <nav class = 'red'>
+            <ul>
+              <li>
+                <a href="/about/" class='hoverlink-color'>About</a>
```

Comment:
You don't need a class to target hover state on an HTML element in CSS. You can just write the CSS rule targeting `a:hover`.